.jshintrc

```diff
@@ -6,7 +6,7 @@
     "forin": true,
     "freeze": true,
     "immed": true,
-    "latedef": true,
+    "latedef": "nofunc",
```

Comment:
I tweaked this, because running `npm test` was otherwise failing on JSHint with the following:

```
$ npm test

> babel-istanbul@0.11.0 pretest /Users/pdehaan/dev/tmp/del/babel-istanbul
> jshint index.js lib/ test/ && ./download-escodegen-browser.sh

lib/collector.js: line 106, col 18, 'makeObj' was used before it was defined.

1 error
npm ERR! Test failed.  See above for more details.
```